### ISSUE TITLE
block comments with prefix

### ISSUE THREAD

**Issue by drahnr** [drahnr/cargo-spellcheck#157]
**Is your feature request related to a particular use-case?**

Currently block comments are not reflown, due the different styles used:

```rust
/**
 *
 *
 */
```
vs

```rust
/**


 */
```

and the broken variant

```rust
/**
 *
 *


 */
```

**Describe the solution you'd like to implement/see implemented**

The prefix should be detected, inconsistencies should be reported as separate errors with a suggestion (bonus).

**Describe alternatives you've considered**

Do not reflow or them. Spellchecking works decently, nlprule results are sub-optimal.

**Comment by drahnr** [comment]
Ref #145
